typings/jsonld.d.ts

```diff
@@ -1,3 +1,9 @@
 declare module 'jsonld' {
-  export function canonize(doc: any): Promise<string>
+  interface Jsonld {
+    canonize: (doc: any) => Promise<string>
+    documentLoader: (url: string, callback: (error: any, data: any) => any) => any
+  }
+  function factory(): Jsonld
+
+  export = factory
```

Comment:
Following [this convo](https://github.com/poetapp/poet-js/pull/150#discussion_r222745581) here...

Maybe try `export = factory | Jsonld`? (or `&` instead of `|`, I never remember which is union and which is intersection)

Reply:
**Union** type `this | that` means *this* **or** *that* will satisfy the type constraint.
**Intersection** type `this & that` means an individual value satisfies the interfaces of both *this* **and** *that*.